python27-sys/Cargo.toml

```diff
@@ -12,6 +12,7 @@ homepage = "https://github.com/dgrunwald/rust-python27-sys"
 repository = "https://github.com/dgrunwald/rust-python27-sys.git"
 license = "Python-2.0"
 authors = ["Daniel Grunwald <daniel@danielgrunwald.de>"]
+links = "python27"
```

Comment:
oh, i think I just realised the misunderstanding - this does not control the linker flags at all, its just a declaration that this crate links to the named native lib for the purposes of signalling to dependent crates looking for a provider of it. again see the doc.crates.io link above.